src/components/Layout/AppName.tsx

```diff
@@ -1,15 +1,31 @@
 import React, { FunctionComponent } from "react";
 import { color, fontSize } from "../../common/styles";
 import { AppText } from "./AppText";
+import { AppMode } from "../../common/hooks/useConfig";
 
-export const AppName: FunctionComponent = () => (
-  <AppText
-    style={{
-      color: color("grey", 0),
-      fontFamily: "inter-bold",
-      fontSize: fontSize(4)
-    }}
-  >
-    Rationally
-  </AppText>
+interface AppName {
+  mode?: AppMode;
+}
+
+export const AppName: FunctionComponent<AppName> = (
+  { mode } = { mode: AppMode.production }
+) => (
+  <>
+    <AppText
+      style={{
+        color: color("grey", 0),
+        fontFamily: "inter-bold",
+        fontSize: fontSize(4)
+      }}
+    >
+      Rationally
+    </AppText>
+    {mode === AppMode.staging ? (
+      <AppText
+        style={{ color: "red", fontWeight: "bold", fontSize: fontSize(3) }}
```

Comment:
`fontWeight: bold` doesn't work when using fonts, need to use `fontFamily: "inter-bold"` instead

Reply:
Didnt know that, thanks!